www/js/main.js

```diff
@@ -50,15 +50,19 @@ angular.module('emission.main', ['emission.main.recent',
   })
   .state('root.main.goals', {
     url: '/goals',
-    templateUrl: 'templates/main-goals.html',
-    controller: 'GoalsCtrl'
-    })
+    views: {
+      'main-goals': {
+        templateUrl: 'templates/main-goals.html',
+        controller: 'GoalsCtrl'
+      }
+    }
+  })
   .state('root.main.signup', {
```

Comment:
I'm going to merge this as-is for now, but instead of a state for the signup, why not use a popup or a popover? That will ensure that the habitica module encapsulates all the habitica information and it is not spread across other places.

e.g. http://ionicframework.com/docs/api/service/$ionicPopup/